mitodl/mitxonline
# mitxonline dashboard crashes when courserun has no upgrade deadline

### What are the relevant tickets?
https://github.com/mitodl/hq/issues/5284

### Description (What does it do?)
mitxonline dashboard crashes when courserun has no upgrade deadline

### How can this be tested?

1. Create a course run with no upgrade deadline defined and course run enrollment for that course run.
2. Visit the dashboard and verify that the page loads correctly.
3. Update the course run to have an upgrade deadline in the future.
4. Visit the dashboard and verify that the page loads correctly.
